archetypes/archetype-lambda/src/main/resources/archetype-resources/template.yaml

```diff
@@ -0,0 +1,11 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ${handlerClassName}Function:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: java8
+      Handler: ${package}.${handlerClassName}::handleRequest
+      Timeout: 60
+      MemorySize: 512
```

Comment:
Maybe we want comments here so people know that they could/should tweak based on their needs (assuming we never make this parameters of the archtype)

Reply:
+1